io/csv/CMakeLists.txt

```diff
@@ -1,18 +1,62 @@
 # Detray library, part of the ACTS project (R&D line)
 #
-# (c) 2021 CERN for the benefit of the ACTS project
+# (c) 2021-2023 CERN for the benefit of the ACTS project
 #
 # Mozilla Public License Version 2.0
 
 # Set the CMake minimum version.
 cmake_minimum_required( VERSION 3.13 )
 
-# Set up the library.
+# Set up the core library.
```

Comment:
Isn't this the ```io_csv``` library?

Reply:
Nomenclature... :stuck_out_tongue: I meant the "core library" for all the I/O libraries. But since I'll be editing this PR anyway, I agree that I should make the language clearer. :wink: